examples/ipam/main.tf

```diff
@@ -0,0 +1,28 @@
+module "vpc" {
+  source = "aws-ia/vpc/awscc"
+
```

Comment:
Each example needs a README so that it doesn't throw an ugly warning in the Terraform Registry like in the screenshot in my other comment, and recommend adding a version like ">= 0.0.1" so that it looks good in the tf-docs, but requires minimal maintenance.

Reply:
I agree that they need a readme! lets talk today more broadly about `source` (see prev convo with @tbulding )